common/source/docs/common-magnetic-interference.rst

```diff
@@ -160,13 +160,8 @@ GPS for yaw
 ===========
 
 ArduPilot supports the use of GPS heading information to reduce malfunctions caused by changes in the magnetic field in certain environments.
-  - `Configure GPS for yaw <https://ardupilot.org/copter/docs/common-gps-for-yaw.html>`__
```

Comment:
sorry, I didn't manage to complete the review of this PR last night.  I think we should put this line re GPS-for-yaw back shouldn't we?